File: .sdk-repos/orchestration-cluster-api-go/examples/system.go
// System and setup operations: initial admin user, usage metrics, and
// system configuration.
package examples

import (
	"context"
	"fmt"

	camunda "github.com/camunda/orchestration-cluster-api-go"
	openapi "github.com/camunda/orchestration-cluster-api-go/client"
)

func createAdminUserExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region CreateAdminUser
	// One-time setup: create the initial administrator on a fresh cluster.
	result, err := client.CreateAdminUser(ctx, *openapi.NewUserRequest("ChangeMe123!", "admin"))
	if err != nil {
		return err
	}
	fmt.Printf("%v\n", result)
	// endregion CreateAdminUser
	return nil
}

func getUsageMetricsExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region GetUsageMetrics
	metrics, err := client.GetUsageMetrics(ctx)
	if err != nil {
		return err
	}
	fmt.Printf("%v\n", metrics)
	// endregion GetUsageMetrics
	return nil
}

func getSystemConfigurationExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region GetSystemConfiguration
	config, err := client.GetSystemConfiguration(ctx)
	if err != nil {
		return err
	}
	fmt.Printf("%v\n", config)
	// endregion GetSystemConfiguration
	return nil
}

File: .sdk-repos/orchestration-cluster-api-php/examples/system.php
<?php

/**
 * Compilable usage examples for system and setup operations.
 */

declare(strict_types=1);

namespace Camunda\Orchestration\Examples;

use Camunda\Orchestration\CamundaClient;

// region CreateAdminUser
/**
 * Create admin user.
 */
function create_admin_user(CamundaClient $client, \Camunda\Orchestration\Api\Model\UserRequest $userRequest): void
{
    $client->createAdminUser($userRequest);
}
// endregion CreateAdminUser

// region GetUsageMetrics
/**
 * Get usage metrics.
 */
function get_usage_metrics(CamundaClient $client, \DateTime $startTime, \DateTime $endTime, ?string $tenantId = null, ?bool $withTenants = false): void
{
    $client->getUsageMetrics($startTime, $endTime, $tenantId, $withTenants);
}
// endregion GetUsageMetrics

// region GetSystemConfiguration
/**
 * System configuration (alpha).
 */
function get_system_configuration(CamundaClient $client): void
{
    $client->getSystemConfiguration();
}
// endregion GetSystemConfiguration
